x-pack/test/alerting_api_integration/spaces_only/tests/actions/builtin_action_types/preconfigured_alert_history_connector.ts

```diff
@@ -0,0 +1,166 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import expect from '@kbn/expect';
+
+import { FtrProviderContext } from '../../../../common/ftr_provider_context';
+import { getTestAlertData, ObjectRemover } from '../../../../common/lib';
+import { AlertHistoryDefaultIndexName } from '../../../../../../plugins/actions/common';
+
+const ALERT_HISTORY_OVERRIDE_INDEX = 'alert-history-not-the-default';
```

Comment:
just had a thought - could we allow the `alert-history` prefix to be configured as well?  That would alleviate my fears of a name collision with a customer's indices.